cluster/charts/rook-ceph/values.yaml.tmpl

```diff
@@ -25,6 +25,9 @@ mon:
   healthCheckInterval: "45s"
   monOutTimeout: "300s"
 
+## Annotations to be added to pod
+podAnnotations: {}
```

Comment:
Could you rename that to just `annotations`.
I don't see where `annotations` would otherwise be useful for the other objects that are created in the Helm chart.